docs-web/installation/installation-with-helm.md

```diff
@@ -352,6 +352,9 @@ The following tables lists the configurable parameters of the NGINX Ingress cont
    * - ``controller.readyStatus.port``
      - The HTTP port for the readiness endpoint.
      - 8081
+   * - ``controller.enableLatencyMetrics``
```

Comment:
do we want to add it now when it might be moved soon under `prometheus`?

Reply:
well, the feature has been there since 1.9.0.